modules/installation-configuration-parameters.adoc

```diff
@@ -1096,13 +1105,33 @@ Additional GCP configuration parameters are described in the following table:
 |Any valid region name, such as `us-central1`.
 
 |`platform.gcp.controlPlaneSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your control plane machines to.
+|The name of the existing subnet where you want to deploy your control plane machines.
 |The subnet name.
 
 |`platform.gcp.computeSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your compute machines to.
+|The name of the existing subnet where you want to deploy your compute machines.
 |The subnet name.
 
+|`platform.gcp.createFirewallRules`
+|Optional. Set this value to `Disabled` if you want to create and manage your firewall rules through the use of network tags. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `roles/compute.networkAdmin` and `roles/compute.securityAdmin` privileges in the host project to perform these tasks automatically. If your service account does not have the `roles/dns.admin` privilege in the host project, it must have the `dns.networks.bindPrivateDNSZone` permission.
+|`Enabled` or `Disabled`. The default value is `Enabled`.
+
+|`platform.gcp.publicDNSZone.project`
+|Optional. The name of the project which contains the public DNS zone. If you set this value, your service account must have the `roles/dns.admin` privilege in the specified project. If you do not set this value, it defaults to `gcp.projectId`.
```

Comment:
Change 'The name of the project which contains...' to 'The name of the project that contains...'.